xc/device_common.ml

```diff
@@ -339,6 +339,8 @@ let demu_restore_path : (_, _, _) format = "/var/lib/xen/demu-resume.%d"
 let var_run_xen_path = "/var/run/xen"
 let qmp_libxl_path = (sprintf "%s/qmp-libxl-%d") var_run_xen_path
 let qmp_event_path = (sprintf "%s/qmp-event-%d") var_run_xen_path
+let efivars_init_path = (sprintf "%s/efi-vars-init-%d.dat") var_run_xen_path
+let efivars_save_path = (sprintf "%s/efivars-%d.dat") var_run_xen_path
```

Comment:
I realize this matches the design, but looking at it now the filenames above seem a bit mismatched. Perhaps this should be `efi-vars-save-%d.dat`?